routes/users.js

```diff
@@ -1,9 +1,15 @@
-const express = require('express');
-const router = express.Router();
+const express = require('express')
+const User = require('../models/user.js')
+const router = express.Router()
+const mongoose = require('mongoose')
 
-/* GET users listing. */
-router.get('/', (request, response, next) => {
-  response.send('respond with a resource');
-});
+router.post('/', ( request, response, next ) => {
+  User.create(request.body, (error, post) => {
```

Comment:
Add space between `(` and `request.body`, as well as between `(`, `)` and parameters in callback function.  Please consider a different name for the parameter `post` - this does not reveal the intent of this parameter (I'm not sure what it does, is intended to represent, or what data it should hold)